test/helpers.lua

```diff
@@ -1,6 +1,7 @@
 local assert = require('luassert')
 local luv = require('luv')
 local lfs = require('lfs')
+local relpath = require('pl.path').relpath
```

Comment:
Do we use pl.path already, or is this a new test dependency?

Reply:
It's included with penlight, so I wouldn't think it's a new dep.